ex2.js

```diff
@@ -0,0 +1,10 @@
+console.log('Script is working')
+
+getReversedString('Hello!'); // '!olleH'
+getReversedString('Arrays'); // 'syarrA'
+
+function getReversedString(stringInput) {
+    stringToArray = stringInput.split("")
+    reversedArray = stringToArray.reverse()
+    return reversedArray.join("")
```

Comment:
remembar about const/let! + this task id working although you should use loops not methods on arrays :)